utils/data.py

```
@@ -0,0 +1,97 @@
+
+def load_data(filename: str) -> List[Any]:
+    """JSON 파일에서 데이터 로드"""
+    ensure_data_dir()
+    filepath = DATA_DIR / filename
+
+    # 파일이 없으면 빈 리스트 반환
+    if not filepath.exists():
+        return []
+
+    try:
+        with open(filepath, "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)